File: proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/MandatoryFieldIsMissing.java
package com.jasonzqshen.familyaccounting.core.exception;

public class MandatoryFieldIsMissing extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = 608695170918084879L;
	public final String _fieldName;

	public MandatoryFieldIsMissing(String fieldName) {
		super(String.format("Mandatory field %s is missing", fieldName));
		_fieldName = fieldName;
	}
}

File: branches/V1.0/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/MandatoryFieldIsMissing.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class MandatoryFieldIsMissing : Exception
    {
        public String FieldName { get; protected set; }
        public MandatoryFieldIsMissing(String fieldName)
            : base(String.Format("Mandatory field {0} is missing", fieldName))
        {
            FieldName = fieldName;
        }
    }
}
